recipes/discourse/webview.js

```diff
@@ -0,0 +1 @@
+module.exports = Ferdium => Ferdium;
```

Comment:
With such an empty webview.js, you will not get any badges showing up. Is that ok?

Reply:
If this is absolutely required I will ping You after I write code to support notification badges.
I'm not a JS programmer and this merge request was just meant to kick-start Discourse support.

Reply:
Sure. But, conversely, as soon as this "shell" is introduced, we usually get folks filing bugs that this feature is not working (lol). I was trying to pre-empt that deluge. If you have any buddies who can contribute for this service (I dont use it, and so I cannot help without a login) - that would be great. If not, I can still approve it, but the bugs will have to stay unsupported till one of the contributors comes with their login to discourse. I hope you understand that.

Reply:
If the above is ok, I can approve the PR